pkg/internal/codegen/parse/util.go

```diff
@@ -30,10 +30,16 @@ import (
 )
 
 const (
-	specReplicasPath   = "specpath"
-	statusReplicasPath = "statuspath"
-	labelSelectorPath  = "selectorpath"
-	jsonPathError      = "invalid scale path. specpath, statuspath key-value pairs are required, only selectorpath key-value is optinal. For example: // +kubebuilder:subresource:scale:specpath=.spec.replica,statuspath=.status.replica,selectorpath=.spec.Label"
+	specReplicasPath    = "specpath"
+	statusReplicasPath  = "statuspath"
+	labelSelectorPath   = "selectorpath"
+	jsonPathError       = "invalid scale path. specpath, statuspath key-value pairs are required, only selectorpath key-value is optinal. For example: // +kubebuilder:subresource:scale:specpath=.spec.replica,statuspath=.status.replica,selectorpath=.spec.Label"
```

Comment:
I did not see these four being used anywhere ?

Reply:
It's gofmt :)